changelogs/fragments/249_fix_junos_command_output.yaml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - Fix junos command RM output for empty config response.
```

What is RM here? junos_command isn't a resource module, and I can't think of anything else it might mean in this context.

If I'm understanding the issue properly, I would suggest instead:
```suggestion
  - junos_command - fix commands that return an empty response (https://github.com/ansible-collections/junipernetworks.junos/issues/249).
```